.github/workflows/on-pull-request-build.yaml

```diff
@@ -0,0 +1,23 @@
+
+name: Build
+on:
+  pull_request:
+
+jobs:
+  tests:
+    name: Run a test build
+    runs-on: ubuntu-20.04
```

Comment:
So why not use `ubuntu-latest` or one of the containers with nodejs already installed?

Something like ubi with node or software collections like what is used for the operator base...
````yaml
    runs-on: ubuntu-latest
    container:
      image: quay.io/repository/fedora/nodejs-16
````

Reference: https://github.com/sclorg/s2i-nodejs-container#nodejs-container-images

Reply:
WOW !! nice :fire:

Reply:
hmm.
quay.io/fedora/nodejs-16
and 
registry.access.redhat.com/ubi8/nodejs-16

do not include yarn :-(

reverting to use
actions/setup-node@v3

Reply:
Gerr, that's sad.

Maybe we can just embed yarn in the repo since yarn is really just a big js file with a node shebang.  (I'm mostly joking, but it is good to keep in mind)